imdb/helpers.py

```diff
@@ -408,7 +408,7 @@ def _valueWithType(tag, tagValue):
 
 # Extra tags to get (if values were not already read from title/name).
 _titleTags = ('imdbindex', 'kind', 'year')
-_nameTags = ('imdbindex')
+_nameTags = 'imdbindex'
```

Comment:
_nameTags is supposed to be an iterable, so I think that the correct fix is:
`_nameTags = ('imdbindex',)`